store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/utils/Visibilities/ColumnVisibility.java

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.parquetstore.utils.visibilities;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * This class is coped from org.apache.accumulo.core.security.ColumnVisibility.
+ */
+public class ColumnVisibility {
```

Comment:
Can we rename this so it makes more sense in the context of the Parquet store, for example ElementVisibility